frontend/app/src/store/balances/exchanges.ts

```diff
@@ -341,6 +352,90 @@ export const useExchangeBalancesStore = defineStore(
       set(exchangeBalances, exchanges);
     };
 
+    const fetchExchangeSavings = async (
+      payload: MaybeRef<ExchangeSavingsRequestPayload>
+    ): Promise<ExchangeSavingsCollection> => {
+      const response = await getExchangeSavings({
+        ...get(payload),
+        onlyCache: true
+      });
+
+      return mapExchangeSavingsCollectionResponse(response);
+    };
+
+    const syncExchangeSavings = async (location: string): Promise<boolean> => {
+      const taskType = TaskType.QUERY_EXCHANGE_SAVINGS;
+
+      const defaults: ExchangeSavingsRequestPayload = {
+        limit: 0,
+        offset: 0,
+        ascending: [false],
+        orderByAttributes: ['timestamp'],
+        onlyCache: false,
+        location
+      };
+
+      const { taskId } = await getExchangeSavingsTask(defaults);
+      const taskMeta = {
+        title: 'test1',
+        message: 'test2'
```

Comment:
😅

Reply:
hardest part of making PR's is write correct sentences in English 😛